templates/ntp.conf.epp

```diff
@@ -101,6 +101,10 @@ ntpsigndsocket <%= $ntp::ntpsigndsocket %>
 <% $ntp::peers.each |$peer| {-%>
 peer <%= $peer %>
 <% } -%>
+# Enable peer group orphan mode?
```

Comment:
Comment should be inside conditional block

Reply:
Isn't it more practical to keep a comment in front of a block so it is still visible in case the block is collapsed in an IDE?
I don't mind moving it into the block, just my thoughts on comment placement.

Reply:
If attribute is not set, you will end up with hanging comment. Also it will cause existing installations to restart named for no reason

Reply:
'ntpd'

Reply:
'ntpd'

Reply:
Ah. Makes sense with templates. The situation could be avoided by encapulating the comment, but that's ugly. It is also redundant if you want to keep the comment in the generated file.